api/docker/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
This is doing a lot! I think it might be doing a bit too much though. The entrypoint is going to run everytime you start up the container, and I'm not sure we need to do all of this every time. I think a good amount of these would be great to do on the initial setup of an enviroment, but not every time.